test/e2e/resources/notebook_instance_lifecycle_config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: NotebookInstanceLifecycleConfig
+metadata:
+  name: $NOTEBOOK_INSTANCE_LFC_NAME
+spec:
+  notebookInstanceLifecycleConfigName: $NOTEBOOK_INSTANCE_LFC_NAME
+  onStart:
+    - content: cGlwIGluc3RhbGwgUElM #pip install PIL
+  onCreate:
+    - content: d2dldCBodHRwOi8vbWlycm9ycy5jdGFuLm9yZy9tYWNyb3MvbGF0ZXgvY29udHJpYi91cHF1b3RlLnppcA0KRU9G
+    # wget http://mirrors.ctan.org/macros/latex/contrib/upquote.zip
```

Comment:
not sure what this is ? is it just a reference

Reply:
The comment is the decoded base 64 string.